apps/prairielearn/src/sync/course-db.js

```diff
@@ -1165,6 +1165,19 @@ async function validateAssessment(assessment, questions) {
               'Cannot specify "maxPoints" or "maxAutoPoints" for a question in an "Exam" assessment'
             );
           }
+
+          const hasSplitPoints =
+            alternative.autoPoints !== undefined ||
+            alternative.maxAutoPoints !== undefined ||
+            alternative.manualPoints !== undefined;
```

Comment:
Nit, feel free to ignore: this expression is repeated in line 1154, do you want to define and use it there?

Reply:
I think I'll leave this as-is, since the error message specifically mentions checking if these values are present. It's clearer to me to check them explicitly than to rely on `hasSplitPoints` which isn't immediately obviously the same check.